driver/lstm_verify_gemm.hpp

```diff
@@ -141,8 +141,8 @@ void RunLSTMForwardGEMMCPUVerify(miopenHandle_t handle,
     // forward emulator
     for(int li = 0; li < numlayer; li++)
     {
-        int hid_shift = li * batch_n * hy_stride;
-        int hx_shift  = li * in_n.at(0) * h_stride;
+        size_t hid_shift = li * batch_n * hy_stride;
+        size_t hx_shift  = li * in_n.at(0) * h_stride;
```

Comment:
[Question] @shurale-nkn Just curious: what would be the the type of `(li * in_n.at(0) * h_stride)` if Tgpu is `float`? I am not aware of the real-life values of li, in_at(0) and h_stride, but operation like `(int * float * int)` may lead to loss of precision due to rounding (I suspect that the type of `int * float` is `float`, so all the computations are happen in float domain).

Reply:
And I am not even asking about case when Tgpu is half ;)

Reply:
> [Question] @shurale-nkn Just curious: what would be the the type of `(li * in_n.at(0) * h_stride)` if Tgpu is `float`? I am not aware of the real-life values of li, in_at(0) and h_stride, but operation like `(int * float * int)` may lead to loss of precision due to rounding (I suspect that the type of `int * float` is `float`, so all the computations are happen in float domain).

...in_n is a vector of int . This is stride in number of elements. And it used only as index

Reply:
@shurale-nkn Ah, I see, my mistake. So this is `(int * int * int)`, which should fit into `int` if the multiply_dims_overflow_assumption is valid:
https://github.com/ROCm/MIOpen/blob/d069763dfd7fa9fd53c3bbfd0513c5c572333c05/src/solver/pooling/forwardNaive.cpp#L125-L136

So you just would like to change the type of the result to size_t due to some reason.
[resolved]

Reply:
Yes, don't worry too much about this file. His code coverage is enough for our solution, there are many problems, but so far it is enough.